src/Symfony/Component/Messenger/Transport/InMemoryTransportFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport;
+
+use Symfony\Component\Messenger\Transport\Serialization\SerializerInterface;
+use Symfony\Contracts\Service\ResetInterface;
+
+class InMemoryTransportFactory implements TransportFactoryInterface, ResetInterface
+{
+    private $createdTransports = [];
```

Comment:
Type of this array should most likely be documented

Reply:
Good point. Added 👍